pkg/route/ingress/ingress.go

```diff
@@ -423,6 +423,14 @@ func (c *Controller) sync(key queueKey) error {
 			continue
 		}
 		for _, path := range rule.HTTP.Paths {
+			pathValue := path.Path
+			if ingress.Annotations[terminationPolicyAnnotationKey] == string(routev1.TLSTerminationPassthrough) && len(path.Path) != 0 {
+				//Ignore the ingress and don't create a route
+				continue
+			}
+			if ingress.Annotations[terminationPolicyAnnotationKey] == string(routev1.TLSTerminationPassthrough) {
+				pathValue = ""
+			}
```

Comment:
This can be simplified as follows:
```suggestion
			pathValue := path.Path
			if ingress.Annotations[terminationPolicyAnnotationKey] == string(routev1.TLSTerminationPassthrough) {
				if len(path.Path) != 0 {
					//Ignore the ingress and don't create a route
					continue
				}
				pathValue = ""
			}
```
However, I think the following is more correct:
```suggestion
			pathValue := path.Path
			if ingress.Annotations[terminationPolicyAnnotationKey] == string(routev1.TLSTerminationPassthrough) {
				if path.Path != "/" {
					// Passthrough is incompatible with paths.
					continue
				}
				pathValue = ""
			}
```
That is, if the ingress has termination type `passthrough`, then skip it if the path isn't "/" and set the path to the empty string if it is "/".